python/pyspark/serializers.py

```diff
@@ -299,14 +300,6 @@ def create_array(s, t):
                 # TODO: don't need as of Arrow 0.9.1
```

Comment:
Do we still need this?

Reply:
I'll check this also

Reply:
No, it's not needed. Tests with Python 2.7 and I manually did a string column conversion to arrow.